sdcm/sct_events/setup.py

```diff
@@ -67,6 +67,8 @@ def start_events_device(log_dir: Optional[Union[str, Path]] = None,
     DbEventsFilter(db_event=DatabaseLogEvent.BACKTRACE, line='Rate-limit: supressed').publish()
     DbEventsFilter(db_event=DatabaseLogEvent.BACKTRACE, line='Rate-limit: suppressed').publish()
     DbEventsFilter(db_event=DatabaseLogEvent.WARNING, line='abort_requested_exception').publish()
+    DbEventsFilter(db_event=DatabaseLogEvent.DATABASE_ERROR, line='ldap_connection - Seastar read failed: '
```

Comment:
it could be a signal of an issue, when seeing these messages without any nemesis, but we should IMHO filter it while during nemesis.

Reply:
The problem is that it happens during any nemesis that restart/shuts down a node, which could be problematic to keep track of.

Reply:
it is not a very widely used test case, nor i'm sure about the impact of this error (maybe users won't be able to create new connections?), but IIRC it started happening because of a bogus nemesis...
i would suggest to not filter it out completely, but maybe lower its priority to be `WARNING` instead of completely ignoring it... WDYT?

Reply:
Hmm, good idea, will lower it to a `WARNING` instead.

Reply:
Done